website_snippet_data_slider/demo/data_slider_demo.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    @license AGPL-3 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+    <template id="data_slider_demo" name="Data Slider Demo"  page="True">
+        <t t-call="website.layout">
+            <div id="wrap" class="oe_structure oe_empty">
+                <div class="container">
+                    <div class="row">
+                        <section class="oe_snippet_body o_data_slider mb32 container text-center"
+                                 data-options="{  
+                                                    &quot;data_model&quot;: &quot;res.partner&quot;,
+                                                    &quot;data_domain&quot;: [[&quot;image&quot;, &quot;!=&quot;, false]],
+                                                    &quot;data_image_field&quot;: &quot;image_medium&quot;,
+                                                    &quot;data_name_field&quot;: &quot;name&quot;,
+                                                    &quot;data_title&quot;: &quot;Partners&quot;,
+                                                    &quot;data_title_tag&quot;: &quot;h1&quot;,
+                                                    &quot;slidesToShow&quot;: 5,
+                                                    &quot;slidesToScroll&quot;: 2,
+                                                    &quot;dots&quot;: true,
+                                                    &quot;infinite&quot;: true,
+                                                    &quot;arrows&quot;: true,
+                                                    &quot;autoplay&quot;: true
```

Comment:
Problem with single quotes is that they don't allow multi-line. I was stuck between a rock and a hard place of having a gargantuan line or using the `&quot;`. I'll switch to one line though, sure